.env

```diff
@@ -0,0 +1,7 @@
+LLAMA_CPP_SERVER_PORT=8090
+LLAMA_CPP_HOST=llama-cpp-server
+LLAMA_CPP_SERVER=http://$LLAMA_CPP_HOST
+LOGDETECTIVE_SERVER_PORT=8080
```

Comment:
Ports here and in the python code should match. Although it's not strictly necessary.